_posts/2022-10-28-multiple-data-source.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "Launch Highlight: Multiple Data Sources"
+authors:
+  - tyarong
+  - zengyan
+  - kroosh
+date:   2022-10-28 00:00:00 -0700
+categories:
+  - community
+---
+
+OpenSearch Dashboards' current architecture works only with a single OpenSearch cluster. To view data in different OpenSearch clusters and to navigate between Dashboards endpoints, users must have a complete OpenSearch with Dashboards stack setup. To solve this challenge, we are excited to announce for release 2.4 the multiple data sources feature. This experimental feature supports multiple data source connections in OpenSearch Dashboards.
+
+###Benefits
+The multiple data sources feature gives users benefits like:
+* **Cost Saving**: Users with hundreds of OpenSearch or Elasticsearch clusters (such as enterprise customers) don't need to create or maintain an equal number of Dashboards servers.
+* **Performance**: Dashboards runs health check against OpenSearch, and in shared-node architecture (that is, OpenSearch and Dashboards run on the same node), CPU and system memory usage can reach 10%, which can cause CPU, memory, network overhead to OpenSearch clusters. Multiple data sources can reduce the number of total running Dashboard instances and trigger health check only to its dedicated metadata storage.
+* **Usability**: Administrators who manage users' access to sensitive data no longer need to manage explicit user credentials across several clusters or configure, install, provision, and administer multiple Dashboards. Users also don't have to navigate across several Dashboards applications to analyze data, combine data, and build reports.
+* **Flexibility**: Users can add, connect, and remove any compatible data sources.
+
+###How can you use the multiple data sources experimental feature?
+
+The first phase of multiple data sources enables Dashboards to query data from multiple compatible OpenSearch endpoints:
+
+* Add or remove data sources that are compatible with Elasticsearch DSL, such as AWS OpenSearch domains, AWS OpenSearch serverless collections, and AWS customer self-managed OpenSearch clusters.
+* Create visualizations comparing time-series data from different index patterns and combine that data in a single dashboard.
+
+![Use case by persona](/assets/media/blog-images/2022-10-28-multiple-data-source/usecase-by-persona.png){: .img-fluid}
```

Comment:
@kristenTian edits to the image


Admin/DevOps
Scenario 1: Should “OSD” be changed to “Dashboards?” Or, is OSD a proper name? 

Scenario 2: Add a comma after “access levels.” Change “tenant level” to “tenant-level.”

Exec/Bus Analyst
Scenario 1: Change “micro-services” to “microservices.” Should there be a space between multiple and (N)? Here’s a suggested rewrite: “I have multiple (N) OpenSearch clusters for N microservices. I want to use single sign-on to one portal to discover, visualize, and analyze data without logging in to a different cluster.”

Scenario 2: Looks good